common/lib/xmodule/xmodule/xml_module.py

```diff
@@ -428,6 +429,13 @@ def add_xml_to_node(self, node):
         """
         # Get the definition
         xml_object = self.definition_to_xml(self.runtime.export_fs)
+        for aside in self.runtime.get_asides(self):
+            if aside.needs_serialization():
+                aside_node = etree.Element("unknown_root", nsmap=XML_NAMESPACES)
+                aside.has_children = False
```

Comment:
@cpennington 
`has_children` attribute is set in [ChildrenModelMetaclass](https://github.com/edx/XBlock/blob/master/xblock/mixins.py#L333) metaclass. `ChildrenModelMetaclass` is metaclass for [HierarchyMixin](https://github.com/edx/XBlock/blob/master/xblock/mixins.py#L355). `XBlock` inherits `HierarchyMixin`:
```
class XBlock(XmlSerializationMixin, HierarchyMixin, ScopedStorageMixin,
RuntimeServicesMixin, HandlersMixin, IndexInfoMixin, ViewsMixin, SharedBlockBase):
```
but `XBlockAside` isn't:
```
class XBlockAside(XmlSerializationMixin, ScopedStorageMixin, 
RuntimeServicesMixin, HandlersMixin, SharedBlockBase):
```
The `self.has_children` check is used in `XmlSerializationMixin.add_xml_to_node` function. It is the part of [xblock](https://github.com/edx/XBlock/blob/master/xblock/mixins.py#L487) project. So how should I correct code in this situation?